src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -155,9 +157,16 @@ function getBlockMapSupportedTags(
 ): Array<string> {
   const unstyledElement = blockRenderMap.get('unstyled').element;
   return blockRenderMap
-    .map((config) => config.element)
-    .valueSeq()
-    .toSet()
+    .reduce((accum, draftBlock, key) => {
+      if (draftBlock.aliasedElements) {
```

Comment:
This is really hard to read. Can you just do

```
var elements = [draftBlock.element];
if (draftBlock.aliasedElements) {
  elements.push(...draftBlock.aliasedElements);
}
```

in a `map` and then flatten?